static/sass/styles.scss

```diff
@@ -535,6 +535,15 @@ summary {
       top: -2rem;
     }
   }
+
+  &--center {
```

Comment:
I don't think you need this anymore

Reply:
Good catch I've cleaned up the whole class